hack/lib/images.bash

```diff
@@ -5,14 +5,15 @@ source "$(dirname "${BASH_SOURCE[0]}")/metadata.bash"
 
 registry_host='registry.ci.openshift.org'
 registry="${registry_host}/openshift"
-export CURRENT_VERSION_IMAGES=${CURRENT_VERSION_IMAGES:-"main"}
+CURRENT_VERSION_IMAGES="release-$(metadata.get project.version)"
```

Comment:
As discussed with @ReToCode , for main branch we'll want something different:
```
export CURRENT_VERSION_IMAGES=${CURRENT_VERSION_IMAGES:-"release-$(metadata.get project.version)"}
```
and then the github action will only set CURRENT_VERSION_IMAGES when running against "main" branch. The value will be "main" then. For other branches it will leave it empty and default to project.version